webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -77,7 +77,7 @@ <h2>{{ report.product }} {{ report.version }} Crash Report [@ {{ report.signatur
               <a href="#details" class="ui-tabs-anchor"><span>Details</span></a>
             </li>
             <li class="ui-state-default ui-corner-top">
-              <a href="#metadata" class="ui-tabs-anchor"><span>Metadata</span></a>
+              <a href="#annotations" class="ui-tabs-anchor"><span>Crash Annotations</span></a>
```

Comment:
This is so much clearer than "metadata".

It does break links and users will be confused for a bit. I could add a note about that, but I think I'm going to wait until someone actually has confusion before doing anything about it.